import_craft.py

```diff
@@ -149,6 +152,12 @@ def generate_parts(self, parts, root, prefabs_dict):
                 self.apply_craft_transformations(part, duplicated)
                 bpy.context.scene.objects.active = duplicated
                 duplicated.name = part['name'] + '_' + part['key']
+
+                # Need to hide fairing if no part at the bottom
+                if 'attN' in part and 'bottom' not in part['attN']:
```

Comment:
would tend to prefer something like 
`if 'bottom' not in part.get('attN', ''):`